electrum/gui/kivy/uix/dialogs/installwizard.py

```diff
@@ -562,6 +565,20 @@ def add_widget(self, widget, index=0):
         else:
             self.crcontent.add_widget(widget, index=index)
 
+    def on_keyboard(self, instance, key, keycode, codepoint, modifier):
+        if key == 27:
+            if self.can_go_back:
```

Comment:
`can_go_back` is a method, so this will always be True

Reply:
ahh no, disregard. there are two of them :)
`self.wizard.can_go_back()` and `self.can_go_back`
is `self.can_go_back` needed?

Reply:
Yes, maybe I should use wizard method.
I'll post changes in a 5 minutes.

Reply:
Removed can_go_back, use wizard method.